bootstrap_storage.go

```diff
@@ -0,0 +1,53 @@
+package unleash
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/Unleash/unleash-client-go/v3/api"
+)
+
+type BootstrapStorage struct {
+	backingStore DefaultStorage
+	Reader       io.Reader
+}
+
+func (bs *BootstrapStorage) Load() error {
+	if len(bs.backingStore.data) == 0 && bs.Reader != nil {
+		dec := json.NewDecoder(bs.Reader)
+		client_features := api.FeatureResponse{}
+		if err := dec.Decode(&client_features); err != nil {
+			return err
+		}
+		bs.backingStore.data = client_features.FeatureMap()
+	}
+	return nil
+}
```

Comment:
Maybe an early return would be even easier to read, not sure. Would camelCase `clientFeatures` at least.

```suggestion
func (bs *BootstrapStorage) Load() error {
	if len(bs.backingStore.data) > 0 || bs.Reader == nil {
		return nil
	}

	dec := json.NewDecoder(bs.Reader)
	clientFeatures := api.FeatureResponse{}
	if err := dec.Decode(&clientFeatures); err != nil {
		return err
	}

	bs.backingStore.data = clientFeatures.FeatureMap()
	return nil
}
```